packages/canary-web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -456,6 +457,7 @@ export class DataTable {
       this.updateSetRowHeight();
     }
     window.addEventListener("resize", this.updateCellHeightsWithDescriptions);
+    window.addEventListener("resize", this.fixCellSelect);
```

Comment:
to save binding 2 event listeners, could these be combined into a single one that calls the 2 methods? the removeEventListener method will need updating too (and it's a bug I think that it wasn't doing a removeEventListener for updateCellHeightsWithDescriptions)